index.js

```diff
@@ -30,6 +34,9 @@ const serializeOne = val => {
   return { [ label ]: xfrm(val) }
 }
 
+const serializeSet =
+  compose(map(String), Array.from)
```

Comment:
* is there a reason everything that would be in a set needs to be a string?
* did you know that `Array.from` accepts a second argument where you can transform the value to be placed into an array? Might be super sweet!
  ```js
   Array.from(new Set(['a', 'b', 3]), String) // [ "a", "b", "3" ]
   // so...
   const serializeSet = s => Array.from(s, String)
   // or...
   const serializeSet = partialRight(Array.from, [String])
  ```

Reply:
Because DynamoDB only supports typed Sets, namely: String Sets, Number Sets, and Binary Sets.  Notice their serialization format in the docs:

![Screen Shot 2019-11-22 at 10 55 49 AM](https://user-images.githubusercontent.com/888052/69440353-af513d80-0d16-11ea-9a17-208170de6d53.png)

While it's **possible** for me to parse each of those into the correct types based on the `SS`, `NS`, or `BS` labels, it's not **feasible** (short of iterating over an entire `Set`) to determine/enforce the "type" of a `Set` in javascript for serialization.  So I thought, let's just stick with String Sets for now (because that's what I need right now), and if it turns out we need more than that, we can always write more code later.

Reply:
Thanks for the explanation! The `SS` makes way more sense now, too